cmake/features.cmake

```diff
@@ -173,6 +173,8 @@ ie_dependent_option(ENABLE_SYSTEM_PROTOBUF "Use system protobuf" OFF
 ie_dependent_option(ENABLE_OV_CORE_UNIT_TESTS "Enables OpenVINO core unit tests" ON "ENABLE_TESTS" OFF)
 ie_option(ENABLE_OPENVINO_DEBUG "Enable output for OPENVINO_DEBUG statements" OFF)
 
+ie_option(ENABLE_CONFORMANCE_PGQL "Enables support of PostgreSQL-based reporting from test tools" OFF)
```

Comment:
Why do we need to have this option on the global level?
Can we remove it from global level and add as conformance specific option? I don't believe that somebody will use it.

Reply:
I'll try. I think I can move it to advanced vars. I need it to control it thru CMake GUI.

Reply:
ie_option => ie_dependent_option.
But if we move it to tests folder, it will be dependent out of box

Reply:
moved to advanced, independent option. please, check